service_desk/i18n/de.po

```diff
@@ -0,0 +1,78 @@
+# Translation of OpenERP Server.
+# This file contains the translation of the following modules:
+# 	* service_desk
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: OpenERP Server 7.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2013-11-14 09:22+0000\n"
+"PO-Revision-Date: 2014-12-10 09:05+0100\n"
+"Last-Translator: openbig.org - Thorsten Vocks <thorsten.vocks@openbig.org>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"X-Generator: Poedit 1.5.5\n"
+
+#. module: service_desk
+#: model:ir.model,name:service_desk.model_project_project
+msgid "Project"
+msgstr "Projekt"
+
+#. module: service_desk
+#: model:ir.model,name:service_desk.model_account_analytic_location
+msgid "account.analytic.location"
+msgstr ""
```

Comment:
Translate it to the same string. Leaving empty doesn't mean the English translation is used.